lib/scarpe/background.rb

```diff
@@ -0,0 +1,13 @@
+class Scarpe
+  module Background
```

Comment:
You'd asked for comment on using a Background module instead of widget

I'm open to other opinions, but I'm okay with Background _not_ being a widget if that's the best way to get it to work.

e.g. We're trying to get full compatibility with Shoes, and most of the time the best way for us is to inherit from the Widget class. However the goal is that the actually Shoes apps run, not to have 1:1 parity as widgets.

Plus background is special in that it relates to either flow, stack or (I believe?) the parent app itself!